src/main/java/io/kuzzle/sdk/core/Collection.java

```diff
@@ -223,6 +223,135 @@ public void onError(JSONObject error) {
   }
 
   /**
+   * Scrolls through specifications using the provided scrollId
+   *
+   * @param scrollId string
+   * @param listener Response callback
+   */
+  public void scrollSpecifications(@NonNull final String scrollId, @NonNull final ResponseListener<JSONObject> listener) {
+    this.scrollSpecifications(scrollId, new Options(), listener);
+  }
+
+  /**
+   * Scrolls through specifications using the provided scrollId
+   *
+   * @param scrollId string
+   * @param options Options Optional parameters
+   * @param listener Response callback
+   */
+  public void scrollSpecifications(@NonNull final String scrollId, final Options options, @NonNull final ResponseListener<JSONObject> listener) {
```

Comment:
For the sake of consistency, shouldn't this method resolve to a SearchResults object instead of a raw JSONObject?

If so, then the corresponding documentation should be updated accordingly.